baybe/searchspace/continuous.py

```diff
@@ -346,6 +346,24 @@ def _drop_parameters(self, parameter_names: Collection[str]) -> SubspaceContinuo
             ],
         )
 
+    @property
+    def is_constrained(self) -> bool:
+        """Return whether the subspace is constrained in any way."""
```

Comment:
property docstring should not start with "return" but be written in the same style as attributes